reduxfx-view/src/main/java/com/netopyr/reduxfx/vscenegraph/impl/patcher/property/SetterAccessor.java

```diff
@@ -17,7 +17,19 @@
     public void set(Consumer<Object> dispatcher, Object node, String name, VProperty vProperty) {
         if (vProperty.isValueDefined()) {
             try {
-                setter.invoke(node, vProperty.getValue());
+                final Object value = vProperty.getValue();
```

Comment:
`Method.invoke` takes a var-arg array of method parameters as argument. However, for this use case the value of vProperty is an array. If the array is put as argument to `invoke` then it uses it as var args so that each element of the array is used as a separate method parameter. 

I haven't found a better way of fixing this. For this usecase only double[] is relevant but I thought that I add code for other types of arrays too.